src/settings/MappingProfiles/MappingProfilesFormContainer/MappingProfilesFormContainer.test.js

```diff
@@ -232,6 +234,34 @@ describe('MappingProfileFormContainer', () => {
           expect(getByText(totalSelected, 'Total selected: 2')).toBeVisible();
         });
 
+        it('should have correct placeholders', () => {
+          expect(transformationFields.length).toEqual(2);
+
+          expect(getByPlaceholderText(transformationFields[0].marcField.container, '900')).toBeInTheDocument();
+          expect(getByPlaceholderText(transformationFields[0].indicator1.container, '0')).toBeInTheDocument();
+          expect(getByPlaceholderText(transformationFields[0].indicator2.container, '0')).toBeInTheDocument();
+          expect(getByPlaceholderText(transformationFields[0].subfield.container, '$a')).toBeInTheDocument();
+
+          expect(queryByPlaceholderText(transformationFields[1].marcField.container, '900')).toBeNull();
+          expect(queryByPlaceholderText(transformationFields[1].indicator1.container, '0')).toBeNull();
+          expect(queryByPlaceholderText(transformationFields[1].indicator2.container, '0')).toBeNull();
+          expect(queryByPlaceholderText(transformationFields[1].subfield.container, '$a')).toBeNull();
+        });
+
+        it('should have correct placeholders after the first transformation is selected and hidden after the selected items filter is unchecked', () => {
+          userEvent.click(screen.getAllByRole('checkbox', { name: 'Select field' })[0]);
+          userEvent.click(screen.getByRole('checkbox', { name: 'Selected' }));
+
+          const transformationFields = getTransformationFieldGroups(); // eslint-disable-line no-shadow
```

Comment:
The way I did it in my tests, is just by reassigning `transformationFields `since it's `let`.  That way, eslint is not complaining.

Reply:
Yeah, have done that, thanks.